portal_objects/metaworkflows/cnv_germline.json

```diff
@@ -223,6 +223,26 @@
         }
       ]
     },
+    {
+      "name": "workflow_BICseq2_add_confidence",
+      "workflow": "dcee73e2-66b2-4f9c-8bd7-43243b7d84fe",
+      "config": {
+        "instance_type": "t3.small",
+        "ebs_size": "2x",
+        "EBS_optimized": true,
+        "spot_instance": true,
+        "run_name": "run_workflow_BICseq2_add_confidence",
+        "behavior_on_capacity_limit": "wait_and_retry"
+      },
```

Comment:
we need to add 

      "custom_pf_fields": {
          "vcf_sv_confidence": {
              "variant_type": "CNV",
              "file_type": "confidence vcf",
              "description": "intermediate file"
          }

Reply:
this fields are used by tibanna to tag the file on the portal, we will probably improve this as well and have more structured tags but for now we need to stick with this